.github/workflows/build.yml

```diff
@@ -35,6 +35,17 @@ jobs:
       - name: Run unit tests
         run: make test
       - name: Run e2e tests
-        run:  make e2e
+        run:  |
+          #!/bin/sh
```

Comment:
nit: this line is redundant: see https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#jobsjob_idstepsshell